packages/cli/src/utils.ts

```diff
@@ -53,48 +53,50 @@ export function getParamJson(jsonFilePath: string): { [_: string]: Options } {
   }
 }
 
-export function getChoicesFromQTNodeQuestion(data: Question, interactive = false): string[] | undefined {
+export function getChoicesFromQTNodeQuestion(
+  data: Question,
+  interactive = false
+): string[] | undefined {
   const option = "option" in data ? data.option : undefined;
   if (option && option instanceof Array && option.length > 0) {
     if (typeof option[0] === "string") {
       return option as string[];
     } else {
       if (interactive) {
-        return (option as OptionItem[]).map(op => op.label);
+        return (option as OptionItem[]).map((op) => op.label);
       } else {
-        return (option as OptionItem[]).map(op => op.cliName ? op.cliName : op.id);
+        return (option as OptionItem[]).map((op) => (op.cliName ? op.cliName : op.id));
       }
     }
   } else {
     return undefined;
   }
 }
 
-export function getSingleOptionString(q: SingleSelectQuestion | MultiSelectQuestion): string | string[] {
+export function getSingleOptionString(
+  q: SingleSelectQuestion | MultiSelectQuestion
+): string | string[] {
   const singleOption = getSingleOption(q);
   if (q.returnObject) {
     if (q.type === NodeType.singleSelect) {
       return singleOption.id;
-    }
-    else {
+    } else {
       return [singleOption[0].id];
     }
-  }
-  else {
+  } else {
     return singleOption;
   }
 }
 
 export function toYargsOptions(data: Question): Options {
   const choices = getChoicesFromQTNodeQuestion(data);
-  if (choices && choices.length > 0 && data.default === undefined) {
-    data.default = choices[0];
-  }
+  // if (choices && choices.length > 0 && data.default === undefined) {
```

Comment:
why comment out this?

Reply:
because there the default value will be changed to lower case and it will cause an error.